language/en/interface.json

```diff
@@ -1041,10 +1041,10 @@
 				"font2": "font 2",
 				"font2_descr": "Stylistic font mainly used for names/buttons/titles",
 				"teamcolors": "Player colors: auto generated ingame",
-				"teamcolors_descr": "Replaces lobby colors with a auto generated color palette based one\n\nNOTE: reloads all widgets because these need to update their colors",
+				"teamcolors_descr": "Replaces lobby colors with a auto generated color palette based on\n\nNOTE: reloads all widgets because these need to update their colors",
```

Comment:
```suggestion
				"teamcolors_descr": "Replaces lobby colors with an autogenerated color palette based one\n\nNOTE: reloads all widgets because these need to update their colors",
```